Core/src/org/sleuthkit/autopsy/casemodule/datasourcesummary/DataSourceSummaryCountsPanel.java

```diff
@@ -39,7 +38,8 @@
     "DataSourceSummaryCountsPanel.FilesByCategoryTableModel.count.header=Count"
 })
 class DataSourceSummaryCountsPanel extends javax.swing.JPanel {
```

Comment:
we can make this class final to prevent the setDataSource(null) from complaining about being overridable as well as remove the logger I will add these changes to PR #6191